cmd/install/install_test.go

```diff
@@ -0,0 +1,58 @@
+package install
+
+import (
+	. "github.com/onsi/gomega"
+	hyperv1 "github.com/openshift/hypershift/api/v1alpha1"
+	"testing"
+)
+
+func TestOptions_Validate(t *testing.T) {
+	tests := map[string]struct {
+		inputOptions Options
+		expectError  bool
+	}{
+		"aws private platform without private creds and region errors": {
```

Comment:
nit: for readability we try to use the "when X it should X" format across the base code, e.g:
When input is aws private platform without creds it should return error.